cafelatte/cafelatte.py

```diff
@@ -0,0 +1,24 @@
+from coffee_time.coffee import Coffee
+import math
+
+
+class Cafelatte(Coffee):
+    """
+    This is a derived class of Coffee
+    """
+
+    def __init__(self, coffee_spoons, sugar, water, milk):
+        super().__init__(coffee_spoons=coffee_spoons, sugar=sugar, water=water)
+        self.milk = milk
+
+    def prepare_coffee(self):
+        cafelatte_quantity = super().prepare_coffee() + self.milk
+        return cafelatte_quantity
+
+    def prepare_time_coffee(self):
+        time_to_prepare = math.pi*math.sin(self.coffee_spoons) + math.cos(self.sugar) + self.water + 2 * self.milk
```

Comment:
This line is too long. It is recommended that you line has a maximum of 79 characters.
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
Some teams strongly prefer a longer line length. For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the nominal line length from 80 to 100 characters.
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
Very good argument! But your line is 111 characters long.

Reply:
Why doesn't PyCharm signal this? :D

Reply:
It does!